docs/protocol/core/v3.0.rst

```diff
@@ -144,26 +144,28 @@ Questions that still need to be resolved
 We solicit feedback on the following area during the RFC period of this first
 draft.
 
- - https://github.com/zarr-developers/zarr-specs/issues/72 to potentially split large metadata documents.
- - extensions and ``must_understand = True`` might be too restrictive. Work a draft implementation with extensions and
-   see how far we can go. List of extensions to implement: 
-   
-      - Boolean
-      - Complex
-      - Datetime
-      - Named dimensions
-      - Awkward arrays
-        
+ - https://github.com/zarr-developers/zarr-specs/issues/72 to potentially split
+   large metadata documents.
+ - extensions and ``must_understand = True`` might be too restrictive. Work a
+   draft implementation with extensions and
```

Comment:
I don't understand the syntax "work a draft implementation"? Do you mean "develop a draft implementation"?

Reply:
My change is was just on the form. @Carreau might have written it?